### PR TITLE
CORE-1484 Allow admins to edit instant launches in nav drawer

### PR DESCRIPTION
**Pull request by ash3rz**
I don't think we're anticipating admins wanting to add more instant launches to the navigation drawer right away, so this doesn't need to be reviewed or merged prior to the release.

Adds the new `Add to / Remove from Navigation Drawer` button for each instant launch:
image

### CODE REVIEW

File: public/static/locales/en/instantlaunches.json [hunk 1]
```diff
@@ -1,10 +1,13 @@
 {
     "addedOn": "Added On",
     "addToDashboard": "Add to Dashboard",
+    "addToNavDrawer": "Add to Navigation Drawer",
     "addedToDashboard": "Instant launch added to the dashboard.",
+    "addedToNavDrawer": "Instant launch added to the navigation drawer.",
     "addMapping": "Add Mapping",
     "addToMapping": "Add Pattern Mapping",
     "cannotAddILToDashboard": "Cannot add IL to the dashboard, a required input parameter is present.",
+    "cannotAddILToNavDrawer": "Cannot add IL to the navigation drawer, a required input parameter is present.",
```
Comment (sri2k1us) [inline public/static/locales/en/instantlaunches.json:10]: Lets avoid acronyms and try to be explicit.

File: src/components/instantlaunches/admin/InstantLaunchList.js [hunk 2]
```diff
@@ -209,8 +271,10 @@ const InstantLaunchList = ({ showErrorAnnouncer }) => {
         },
     });
 
-    const isLoading = allILs.isLoading || dashboardILs.isLoading;
-    const isError = allILs.isError || dashboardILs.isError;
+    const isLoading =
+        allILs.isLoading || dashboardILs.isLoading || navDrawerILs.isLoading;
```
Comment (sri2k1us) [inline src/components/instantlaunches/admin/InstantLaunchList.js:275]: Should we also check for `removeFromDrawerMutation` and `addToNavDrawerMutation` status ?

File: src/serviceFacades/instantlaunches.js [hunk 3]
```diff
@@ -221,6 +244,31 @@ export const removeFromDashboardHandler = async (id) => {
...
+            value !== instantLaunchNavDrawer
+    );
+
+    if (filtered.length > 0) {
+        return await resetInstantLaunchMetadata(id, filtered);
+    }
+
+    return new Promise((resolve, reject) => resolve(ilMeta));
+};
+
```
Comment (sri2k1us) [inline src/serviceFacades/instantlaunches.js:271]: While this works, I wonder if we can simplify (and standardize) by using react-queries - https://react-query.tanstack.com/guides/dependent-queries#_top
Same for `addToNavDrawer` function as well.

File: src/components/instantlaunches/admin/InstantLaunchList.js (hunk 2 above)
Comment (ash3rz) [reply src/components/instantlaunches/admin/InstantLaunchList.js:275]: Yes - good catch. There were others missing too.  I'll add those.

File: src/serviceFacades/instantlaunches.js (hunk 3 above)
Comment (ash3rz) [reply src/serviceFacades/instantlaunches.js:271]: I agree - I saw some opportunities to refactor for consistency with what we're doing in other components, but I also wanted to keep the changes short for this PR.  I can definitely do some refactoring in a separate PR.

**Pull request merged**